src/balatro_gym/game/scoring.py

```diff
@@ -91,13 +91,19 @@ def _get_flush(hand: Sequence[PlayingCard]) -> Sequence[PlayingCard]:
         if card.enhancement is not None:
             counter.update(card.enhancement.get_suit(card))
         else:
-            counter.update([card.suit])
+            counter.update(card.suit)
+
+    most_common_tup = counter.most_common(1)
```

Comment:
nit: what does tup stand for? suggestion: `most_common_suit`

Reply:
tup == tuple, but that's actually not even accurate because `counter.most_common()` returns a list of tuples. Updated naming to increase clarity